wgpu-hal/src/gles/wgl.rs

```diff
@@ -570,6 +587,43 @@ impl crate::Instance for Instance {
     }
 }
 
+impl super::Adapter {
+    /// Creates a new external adapter using the specified loader function.
+    ///
+    /// # Safety
+    ///
+    /// - The underlying OpenGL ES context must be current.
+    /// - The underlying OpenGL ES context must be current when interfacing with any objects returned by
+    ///   wgpu-hal from this adapter.
+    /// - The underlying OpenGL ES context must be current when dropping this adapter and when
+    ///   dropping any objects returned from this adapter.
```

Comment:
On wgl I think it is not an `ES` context?

Reply:
We definitely request it via Glutin and get one:

```console
$ RUST_LOG=trace cargo r --example raw-gles
    Finished `dev` profile [unoptimized + debuginfo] target(s) in 0.41s
     Running `target\debug\examples\raw-gles.exe`
Initializing external GL context
Picked a config with 16 samples
Hooking up to wgpu-hal
[2024-08-29T21:19:06Z DEBUG wgpu_hal::gles::adapter] Vendor: Intel
[2024-08-29T21:19:06Z DEBUG wgpu_hal::gles::adapter] Renderer: Intel(R) Arc(TM) A770 Graphics
[2024-08-29T21:19:06Z DEBUG wgpu_hal::gles::adapter] Version: OpenGL ES 3.0 - Build xxxx
[2024-08-29T21:19:06Z DEBUG wgpu_hal::gles::adapter] SL version: OpenGL ES GLSL ES 3.00 - Build xxxx
[2024-08-29T21:19:06Z DEBUG wgpu_hal::gles::adapter] Supported GL Extensions: {
...
}
```

Reply:
I see. I guess the WGL backend can use either type? Since iiuc the non-external WGL one isn't an ES context.